rio_rgbify/encoders.py

```diff
@@ -32,31 +33,29 @@ def data_to_rgb(data, baseval, interval):
     datarange = data.max() - data.min()
 
     if _range_check(datarange):
-        raise ValueError('Data of {} larger than 256 ** 3'.format(datarange))
+        raise ValueError("Data of {} larger than 256 ** 3".format(datarange))
```

Comment:
@vincentsarago rio-rgbify should follow Brett Slatkin's [root exception principle](https://books.google.com/books?id=bTUFCAAAQBAJ&pg=PA185&lpg=PA185&dq=slatkin+root+exception&source=bl&ots=Gag9ZgpfOF&sig=ACfU3U1MVgNBciy-uMYvvf4VOFITIEZbWg&hl=en&sa=X&ved=2ahUKEwidkaHVianmAhXXpJ4KHVJaDhMQ6AEwAnoECAoQAQ#v=onepage&q=slatkin%20root%20exception&f=false) and raise a new exception instead of ValueError.

Reply:
@sgillies 👌, TBH, this PR is just CI+format related I didn't really looked at the code itself